test/integration/client-side-encryption/client_side_encryption.prose.14.decryption_events.test.ts

```diff
@@ -75,11 +75,14 @@ describe('14. Decryption Events', metadata, function () {
       keyId: keyId,
       algorithm: 'AEAD_AES_256_CBC_HMAC_SHA_512-Deterministic'
     });
-    // Copy ``ciphertext`` into a variable named ``malformedCiphertext``.
-    // Change the last byte to 0. This will produce an invalid HMAC tag.
+    // Copy ``ciphertext`` into a variable named ``malformedCiphertext``. Change the
+    // last byte to a different value. This will produce an invalid HMAC tag.
     const buffer = Buffer.from(cipherText.buffer);
-    buffer.writeInt8(0, buffer.length - 1);
+    const lastByte = buffer.readInt8(buffer.length - 1);
+    const replacementByte = lastByte === 0 ? 1 : 0;
+    buffer.writeUint8(replacementByte, buffer.length - 1);
```

Comment:
```suggestion
    buffer.writeUInt8(replacementByte, buffer.length - 1);
```

Pretty sure it's a capital `I`

Reply:
That's odd - how did it pass then?

Reply:
Both work:
```typescript
> let b = Buffer.from([1,2,3])
undefined
> b.writeUint8(0)
1
> b.writeUInt8(1)
1
```

Reply:
My guess is that maybe they support both because JS's Uint8Array is spelled with lowercase i?

Reply:
wow TIL!

Reply:
Yup :wink: https://github.com/nodejs/node/commit/5864fca7bc96b